client/src/lit/compat/lazy-bcd-table.js

```diff
@@ -0,0 +1,85 @@
+import { LitElement, html } from "lit";
+import { createComponent } from "@lit/react";
+import React from "react";
+import { BCD_BASE_URL } from "../../env.ts";
+import "./bcd-table.js";
+
+class LazyBcdTable extends LitElement {
+  static properties = {
+    _id: {},
+    query: {},
+    locale: {},
+    compat: { state: true },
+    error: { state: true },
+    loading: { state: true },
+  };
+
+  constructor() {
+    super();
+    this._id = "";
+    this.query = "";
+    this.locale = "";
+    this.compat = null;
+    this.error = null;
+    this.loading = false;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.loading = true;
+  }
+
+  /**
+   * @param {Map<string, any>} changedProperties
+   * @returns {Promise<void>}
+   */
+  async update(changedProperties) {
+    super.update(changedProperties);
+    if (changedProperties.has("query")) {
+      await this.fetchData(this.query);
+    }
+  }
```

Comment:
Reckon this might be the source of the blank table bug: we should use a Task here: https://lit.dev/docs/data/task/ you can see an example of it being used in the play runner: https://github.com/mdn/yari/blob/84f797a3364777be2e4b21346f68ce66689105fc/client/src/lit/play/runner.js#L92

Reply:
Thanks, fixed in 7d169128f3.